src/main/kotlin/frc/team5499/frc2018Kotlin/utils/Utils.kt

```diff
@@ -21,7 +29,7 @@ object Utils {
     }
 
     fun encoderTicksToInches(ticks: Int): Double {
-        return (Constants.WHEEL_CIR / Constants.ENCODER_TICKS_PER_ROTATION) * ticks
+        return (Constants.WHEEL_CIR / Constants.ENCODERS_TICKS_PER_ROTATION) * ticks
```

Comment:
change this back to `ENCODER_TICKS_PER_ROTATION`

Reply:
ok